tools/redcap2linst.php

```diff
@@ -167,6 +167,10 @@ function outputFiles(string $outputDir, array $instruments)
 {
     foreach ($instruments as $instname => $instrument) {
         $fp = fopen("$outputDir/$instname.linst", "w");
+        fwrite($fp, "table{@}$instname\n");
+        // FIXME: There should be a title{@} tag for it to be a valid linst
+        // that the instrument manager can recognize, but there doesn't seem
+        // to be any way to derive that from the redcap data dictionary csv?
```

Comment:
@driusan when i ran into this issue, i took the table name, split it on the delimiter and changed every first letter to uppercase. would this work for you?